Documentation/ceph-backup.md

```diff
@@ -0,0 +1,483 @@
+---
+title: Back Up
+weight: 2200
+indent: true
+---
+
+# Back Up
+
+Back Up of Ceph helps you recover your data when a disaster occurs.
+
+## Requirements
+
+The following requirements are needed:
+
+1. Rook Ceph [PVC-based Cluster](ceph-cluster-crd.md#pvc-based-cluster).
+2. [Velero](https://velero.io/) + [Restic](https://velero.io/docs/v1.5/restic/)
+
+## Deploy of a Ceph PVC-based Cluster
+
+### Deploy the operator
+
+The first step is to deploy the Rook operator. Check that you are using the [example yaml files](https://github.com/rook/rook/blob/{{ branchName }}/cluster/examples/kubernetes/ceph) that correspond to your release of Rook. For more options, see the [examples documentation](ceph-examples.md).
+
+```console
+cd cluster/examples/kubernetes/ceph
+kubectl create -f common.yaml
+kubectl create -f operator.yaml
+
+## verify the rook-ceph-operator is in the `Running` state before proceeding
+kubectl -n rook-ceph get pod
+```
+
+You can also deploy the operator with the [Rook Helm Chart](helm-operator.md).
+
+### Create a Rook Ceph PVC-based Cluster
```

Comment:
Instead of describing how to setup the PVC-based cluster, we should have a link to another topic that says how to deploy it, then this topic just talks about the backup and restore steps. 

Since we don't have good documentation around PVC-based clusters, perhaps you can add a paragraph in [this topic](https://rook.github.io/docs/rook/v1.4/ceph-cluster-crd.html#pvc-based-cluster) to make it more clear how to deploy.